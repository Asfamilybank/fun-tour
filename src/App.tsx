import { useState } from 'react'

const App = () => {
  const [count, setCount] = useState(0)

  return (
    <div>
      <header>
        <p className="m-2">Hello Fun Tour</p>
        <p>
          <button type="button" onClick={() => setCount((count) => count + 1)}>
            count is: {count}
          </button>
        </p>
      </header>
    </div>
  )
}

export default App
