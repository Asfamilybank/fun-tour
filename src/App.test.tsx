import { render } from '@testing-library/react'
import App from 'App'

it('test right', () => {
  const { asFragment } = render(<App />)

  expect(asFragment).matchSnapshot()
})
