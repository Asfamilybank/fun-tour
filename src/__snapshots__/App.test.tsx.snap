// Vitest Snapshot v1

exports[`test right 1`] = `[Function]`;
